Add `Vocaloid Mapping Contest 2 Results` news post (#11213)
ppy/osu-wiki: wiki/Contests/VMC/2/en.md

| Judging | 2024-03-08 to 2024-04-05 |
| Results livestream | 2024-04-07 @ 18:00 UTC |

> The results were streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. You can view the recordings for all of the contests on the [*Vocaloid Appreciation Team* YouTube channel](https://www.youtube.com/@VocaloidAppreciationTeam).

Reviewer: avoiding pronouns in the wiki

```suggestion
The results were streamed live on the [*Vocaloid Appreciation Team* Twitch account](https://www.twitch.tv/vocaloidappreciationteam) with commentary. Recordings for all of the contests are available on the [*Vocaloid Appreciation Team* YouTube channel](https://www.youtube.com/@VocaloidAppreciationTeam).
```

| 14 (Miku ver.) | Hatsune Miku | π | 4:55 | 128 | [.osz](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%2014.osz) |
| sigh | Kagamine Rin | iroha(sasaki) | 4:08 | 148 | [.osz](https://assets.ppy.sh/artists/399/Songs/iroha(sasaki)%20-%20sigh.osz) |

## Results 

### osu!

> Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44)!

Reviewer: the uses a more encyclopedic tone

```suggestion
Detailed scores, judge comments, and final placements are [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91885b2923e167f79fd44).
```

| Placement | User | Anonymised beatmap | Badge prize |
| :-: | :-- | :-- | :-- |
| 1st | [Nuolong](https://osu.ppy.sh/users/4732455) | world language | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
| 2nd | [yaspo](https://osu.ppy.sh/users/4945926) | how prove | - |
| 3rd | [Meijiro McQueen](https://osu.ppy.sh/users/11555612) | having mental | - |
| 4th | [celerih](https://osu.ppy.sh/users/4696296) | president summer | - |
| 5th | [Nijika Ijichi](https://osu.ppy.sh/users/10964252) | anything practice | - |

### osu!taiko

> Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91aedb2923e167f79fe92)!

Reviewer: ```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91aedb2923e167f79fe92).
```

| Placement | User | Anonymised beatmap | Badge prize |
| :-: | :-- | :-- | :-- |
| 1st | [uone](https://osu.ppy.sh/users/5321719) | try storm | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
| 2nd | [Nifty](https://osu.ppy.sh/users/4956097) | naturally if | - |
| 3rd | [-Kazuha](https://osu.ppy.sh/users/29978316) | birthday slight | - |

### osu!catch

> Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91b13b2923e167f79feec)!

Reviewer: ```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91b13b2923e167f79feec).
```

| Placement | User | Anonymised beatmap | Badge prize |
| :-: | :-- | :-- | :-- |
| 1st | [Dika312](https://osu.ppy.sh/users/741613) | happily under | ![](/wiki/Contests/VMC/2/img/vmcwinnerbadge.png) |
| 2nd | [Unlucky_w](https://osu.ppy.sh/users/4820793) | curve blue | - |
| 3rd | [Cipse](https://osu.ppy.sh/users/7928880) | flow late | - |

### osu!mania

> Detailed scores, judge comments, and final placements are available [HERE](https://mappersguild.com/contests/results?contest=65d91a82b2923e167f79fe20)!

Reviewer: ```suggestion
Detailed scores, judge comments, and final placements are available [available on the Mappers' Guild website](https://mappersguild.com/contests/results?contest=65d91a82b2923e167f79fe20).
```
